src/cohere/manually_maintained/cache.py

```diff
@@ -0,0 +1,18 @@
+class CacheMixin:
+    # Optional caching mixin with time-to-live. If cachetools is not installed, caching is disabled.
+    cache = None
+    try:
+        from cachetools import TTLCache
+
+        cache = TTLCache(maxsize=100, ttl=60)
+    except ImportError:
+        pass
+
+    def cache_get(self, key):
```

Comment:
will these be exposed to user on the client object?

Reply:
good catch!